test/test.cc

```diff
@@ -8,6 +8,11 @@
 #include "midgard/sequence.h"
 #include "mjolnir/graphtilebuilder.h"
 
+#include <boost/algorithm/string.hpp>
+#include <boost/property_tree/ptree.hpp>
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
```

Comment:
these two are already included via `test.h`

Reply:
gmock is needed for the P2_MATCHER macro, but this can be changed to `#include <gmock/gmock-matchers.h>`, maybe it should be be extracted to a new header.
gtest is indeed not need.